Makefile

```diff
@@ -171,19 +177,23 @@ install-member-agent-helm: install-hub-agent-helm e2e-hub-kubeconfig-secret
 	kubectl delete pod --all -n fleet-system
 
 build-e2e:
-	go test -c ./test/e2e
+	go test -c ./test/e2e/v1alpha1
 
 run-e2e: build-e2e
-	KUBECONFIG=$(KUBECONFIG) HUB_SERVER_URL="https://$$(docker inspect $(HUB_KIND_CLUSTER_NAME)-control-plane --format='{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}'):6443" ./e2e.test -test.v -ginkgo.v
+	KUBECONFIG=$(KUBECONFIG) HUB_SERVER_URL="https://$$(docker inspect $(HUB_KIND_CLUSTER_NAME)-control-plane --format='{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}'):6443" ./v1alpha1.test -test.v -ginkgo.v
 
-.PHONY: creat-kind-cluster
-creat-kind-cluster: create-hub-kind-cluster create-member-kind-cluster install-helm
+.PHONY: create-kind-cluster
+create-kind-cluster: create-hub-kind-cluster create-member-kind-cluster install-helm
 
 .PHONY: install-helm
 install-helm:  load-hub-docker-image load-member-docker-image install-member-agent-helm
 
 .PHONY: e2e-tests
-e2e-tests: creat-kind-cluster run-e2e
+e2e-tests: create-kind-cluster run-e2e
+
+.PHONY: e2e-tests-v1beta1
+e2e-tests-v1beta1:
```

Comment:
I had a discussion with Ryan before. The better way is to have one setup for both v1alpha1 &v1beta1 APIs tests and the controller will watch both APIs. 

We will run two sets of test cases:
    - one set of v1alpha1 CRP test cases
    - one se of v1beta1 CRP test cases.

Reply:
As mentioned in chat, I was not aware of this discussion at all.

Reply:
I had a discussion with Ryan. Due to the different in setup + the complication of enabling parallelization in v1alpha1 code, it might make more sense to keep them separate for now.

Reply:
I'm ok with creating a new setup for the v1beta1 APIs if the current setup cannot meet our requirement and you've already made the new setup work. 

My concern is how  to deal with v1 API in the future. My idea is to make use of the v1beta1 environment and create another set of v1 test cases. 

WDYT? if we all agree on this, the code structure needs to be updated to reflect that, renaming the existing "e2e" package name to v1alpha1e2e and use the "e2e" for the v1beta1 + v1 (in the future).

Reply:
Discussed offline. Will tweak the organization but leaves the env. separate as originally proposed.